src/cgutils.cpp

```diff
@@ -1381,6 +1381,16 @@ static Value *emit_arraysize(Value *t, jl_value_t *ex, int dim, jl_codectx_t *ct
     return emit_arraysize(t, dim);
 }
 
+static Value *emit_arrayflags(Value *t, jl_codectx_t *ctx)
+{
+    Value *addr = builder.CreateStructGEP(
+#ifdef LLVM37
+                            nullptr,
```

Comment:
???
am I blind ? The CI should never see this line since it runs on 3.3 ?

but more importantly the exact same things is done just above ? Am I missing something really obvious ?